apps/parsley/package.json

```diff
@@ -32,7 +32,7 @@
     "verify-backend": "./scripts/verify-backend.sh",
     "storybook": "storybook dev -p 6006",
     "storybook:build": "env-cmd -e local -r .env-cmdrc.local.json storybook build",
-    "test": "yarn --cwd='../..' test --selectProjects parsley",
+    "test": "yarn --cwd='../..' test --selectProjects parsley --watchAll=false",
```

Comment:
Changed this because I was unable to run specific test like `yarn test useParsleySetting`, it would just run all the tests. Adding this allowed me to run specific tests

Reply:
Could you add this to the root `test` script instead? I imagine it'd be helpful for every workspace.

Reply:
And good catch! 😁